fili-core/src/main/java/com/yahoo/bard/webservice/sql/helper/DatabaseHelper.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.helper;
+
+import java.sql.Connection;
+import java.sql.JDBCType;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+
+/**
+ * Various functions to help with interacting with a database.
+ */
+public class DatabaseHelper {
+    /**
+     * Private constructor - all methods static.
+     */
+    private DatabaseHelper() {
+    }
+
+    /**
+     * Finds the name of the Timestamp column in a sql database.
+     *
+     * @param connection  The connection to the database.
+     * @param tableWithSchema  The name of the table with schema (i.e. "public"."table").
+     *
+     * @return the name of the timestamp column.
+     *
+     * @throws SQLException if failed while reading database.
```

Comment:
Yea, I have yet to test different date types in SQL so I've limited it to only being a timestamp, but was planning on trying out `CAST` later.

I've logged the column name/documented the exception for now, but I agree it might be best to just remove this, keep the requirement for a timestamp column, and require that the column is explicitly specified.